File: variables.js
/* Variables (var, let, const) */
// comentario simple

let nombre = ""
let apellido = ''
let nombreCompleto = `contenido`

let edad = 50;
let saldo = 14.50;
let grados = -10.5;
let posicion = -5;

let activo = true || false;

let direccion;

let usuarios = null;

let persona = {
    nombre: '',
    apellido: ''
}

console.log(persona.nombre);
console.log(persona['nombre']);

let frutas = ["Pera", "Manzana", "Uva"];
//let nombres = new Array("Pedro", "Juan", "Miguel");


console.log(frutas[0])


function sumar(a, b){
    return a + b;
}

let restar = function(a, b){
    return a - b;
}

function multiplicar(a, b){
    
}

File: variables.py
""" Variables """
# comentario simple

nombre = ""
apellido = ''
nombre_completo = '''contenido'''

edad = 50
saldo = 14.50
grados = -10.5
posicion = -5

activo = True | False

direccion = None

usuarios = None

persona = {
    "nombre": "",
    "apellido": ""
}

print(persona['nombre'])


# List
frutas = ["Pera", "Manzana", "Uva"]

# Tuples
nombres = ("Pedro", "Juan", "Miguel")

# Set
status = { "activo", "inactivo", "cancelado", "suspendido", "completado"} 


print(frutas[0])

print(nombres[2])

print(status)

def sumar(a, b):
    return a + b 

restar = lambda a, b : a - b

def multiplicar(a, b):
    pass 